src/transformers/models/auto/image_processing_auto.py

```diff
@@ -127,6 +127,7 @@
             ("poolformer", ("PoolFormerImageProcessor",)),
             ("pvt", ("PvtImageProcessor",)),
             ("pvt_v2", ("PvtImageProcessor",)),
+            ("qwen2_5_vl", ("Qwen2_5_VLImageProcessorFast", "Qwen2_5_VLImageProcessorFast")),
```

Comment:
if the image processor is the same we only need this: 
```suggestion
            ("qwen2_5_vl", ("Qwen2_5_VLImageProcessorFast", "Qwen2VLImageProcessorFast")),
```
But if some stuffs are different LGTM

Reply:
Yes, the image processor between Qwen2-VL and Qwen2.5-VL is exactly same, they only differ on processor. I think we can remove the image processor porting from Qwen2.5-VL modular.